Create 2022-05-23.md (#217)
solid/authentication-panel: meetings/2022-05-23.md

* Use panel chat and repository. Queue in call to talk.

### Participation and Code of Conduct
* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
* [Solid Code of Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md)
* If this is your first time, welcome! please introduce yourself.

### Scribes
* Pavlik  

---

## Agenda

### Open PRs

https://github.com/solid/solid-oidc/pulls

* [primer: use correct token in request flow (steps 7 & 8) #170](https://github.com/solid/solid-oidc/pull/170) ✅
* [make webid scope required for DCR #171](https://github.com/solid/solid-oidc/pull/171)

### Issues

https://github.com/solid/solid-oidc/issues

* [Autoexpiry of DCR-registered clients #167](https://github.com/solid/solid-oidc/issues/167)

## Minutes

### Require "webid" scope for OIDC Dynamic Client Registration #168

> * AC: Should webid be a must? Let's say you initiate DCR and specify that you want scopes `openid email`. Later on you want to use `client_id` and `client_secret` want to use an additional scope. I think it would give an invalid scope.

Reviewer: ```suggestion
* AC: Should webid be a must? Let's say you initiate DCR and specify that you want scopes `openid email`. Later on you want to use `client_id` and `client_secret` with an additional scope. An OP would give an invalid scope error.
```
